### PR TITLE
Improvements and fixes before initial release

### PR DESCRIPTION
**Pull request by mecha**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,8 +13,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^5.3 | ^7.0",
-        "dhii/data-identifiable-interface": "^0.1.0",
-        "dhii/validation-interface": "^0.1.0"
```
Comment (XedinUnknown) [inline composer.json:17]: Why is validation interface package not a requirement?

Comment (mecha) [reply composer.json:17]: Nothing in this package needs validation interfaces.

File: src/Factory/NonceFactoryInterface.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+namespace RebelCode\WordPress\Nonce\Factory;
+
+use RebelCode\WordPress\Nonce\NonceInterface;
+
+/**
+ * Something that can create nonce instances.
+ *
+ * @since [*next-version*]
+ */
+interface NonceFactoryInterface
```
Comment (XedinUnknown) [inline src/Factory/NonceFactoryInterface.php]: Not gona extend `FactoryInterface`?

Comment (mecha) [reply src/Factory/NonceFactoryInterface.php]: Good catch. Didn't notice this.

**Review by XedinUnknown — APPROVED** [review]

**Pull request merged**